controllers/containerimagebuild_controller.go

```diff
@@ -229,3 +237,14 @@ func (r *ContainerImageBuildReconciler) updateResourceStatus(ctx context.Context
 	}
 	return nil
 }
+
+// This logs the underlying error from a build when the display channels inside builder.embedded have not yet been initialized.
+func logUnknownInstructionError(log logr.Logger, err error) {
+	if unwrappedError := errors.Unwrap(err); unwrappedError != nil {
```

Comment:
Does `errors.Cause(errors.Cause(err))` not accomplish the same thing here w/o the nil check? Just curious.

Reply:
So `err` in this case is a `fmt.wrapError` which wraps an error that we need the `Cause` of. It's unfortunately an issue of two different libraries. In actuality `errors.Unwrap` is just a passthrough for the stdlib errors library:
https://github.com/pkg/errors/blob/614d223910a179a466c1767a985424175c39b465/go113.go#L37

Maybe I should codify that here and call `stderrors.Unwrap` as well?